lib/ansible/runner.rb

```diff
@@ -408,34 +413,42 @@ def wait_for_listener_start(listener)
         end
       end
 
-      def python_path
-        @python_path ||= [manageiq_venv_path, *ansible_python_paths].compact.join(File::PATH_SEPARATOR)
+      VENV_ROOT = "/var/lib/manageiq/venv".freeze
+
+      def venv_python_path
+        return @venv_python_path if defined?(@venv_python_path)
+
+        @venv_python_path = Dir.glob(File.join(VENV_ROOT, "lib/python*/site-packages")).first
```

Comment:
Did you want to change the `*` to `3.12` before merge?

Reply:
yes, let me do that.

Reply:
Done